bang/views.py

```diff
@@ -14,29 +14,9 @@
 # Live2D
 
 def live2d(request, pk, slug=None):
-    ajax = request.path_info.startswith('/ajax/')
-    context = ajaxContext(request) if ajax else getGlobalContext(request)
-    context['ajax'] = ajax
-
-    queryset = models.Card.objects.filter(id=pk, live2d_model_pkg__isnull=False)
+    queryset = models.Card.objects.select_related('associated_costume').filter(id=pk, associated_costume__isnull=False)
     the_card = get_one_object_or_404(queryset)
-
-    context['page_title'] = u'{}: {}'.format('Live2D', unicode(the_card))
-    context['item'] = the_card
-    context['package_url'] = get_image_url_from_path(the_card.live2d_model_pkg)
-
-    # Work around jQuery behaviour; details in pages/live2dviewer.html.
-    if ajax:
-        context['late_js_files'] = LIVE2D_JS_FILES
-        context['danger_zone'] = 220
-    else:
-        context['js_files'] = LIVE2D_JS_FILES
-        context['danger_zone'] = 100
-
-    context['extends'] = 'base.html' if not context['ajax'] else 'ajax.html'
-    context['canvas_size'] = (562, 562) if context['ajax'] else (1334, 1000)
-
-    return render(request, 'pages/live2dviewer.html', context)
+    return redirect(the_card.associated_costume.viewer_url)
```

Comment:
You removed the it from the enabled pages, so this is basically dead code. We can keep the URL and redirect, but tbh, I don't think it stayed up long enough to worry about dead links. I can't find any link shared on Twitter, Facebook, Tumblr, Reddit, VK or just Google, so I think we can delete it.